Ana_Mamani/practice10.py

```diff
@@ -0,0 +1,16 @@
+# Practice 10
+# author Ana Mamani Zenteno
+# since 10/05/2018
+
+def reads_a_string(cadena):
```

Comment:
Please be careful with the language that you are using try to use all in English